templates/Image_Stitching.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
<p>I would change <code>image_dir</code> to <code>segmentation_dir</code> , and same below, to further emphasize what's being selected.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/702/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>